source/index.html

```diff
@@ -1,12 +1,140 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Седона</title>
   </head>
-  <body>
+  <body class="page__body">
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
+    <header class="page-header">
+      <a class="page-header__logo">
+        <img src="img/logotype-mobile.svg" width="102" height="84" alt="Логотип сайта «Седона»">
+      </a>
+      <nav class="main-nav">
+
+        <button class="main-nav__toggle" type="button">
+          <span class="visually-hidden">Открыть меню</span>
+        </button>
+
+        <div class="main-nav__wrapper">
+          <ul class="main-nav__list site-list">
+            <li class="site-list__item site-list__item--current">
+              <a>Главная</a>
+            </li>
+            <li class="site-list__item">
+              <a href="catalog.html">Фото и видео</a>
+            </li>
+            <li class="site-list__item">
+              <a href="form.html">Форма отзыва</a>
+            </li>
+            <li class="site-list__item">
+              <a href="#">HTML Academy</a>
+            </li>
+          </ul>
+        </div>
+
+      </nav>
+    </header>
+
+    <main class="page-main">
+
+      <h1 class="visually-hidden">Седона - бронирование гостиниц</h1>
+
+      <section class="intro">
+        <div class="intro__wrapper">
+          <h2 class="visually-hidden">Приветствие</h2>
+          <p class="intro__slogan">Седона &mdash; небольшой городок в Аризоне, заслуживающий большего!</p>
+          <p class="intro__description">Рассмотрим 5 причин, по которым Седона круче, чем Гранд Каньон!</p>
+        </div>
+      </section>
+
+      <section class="reasons">
+        <div class="reasons__wrapper">
+          <h2 class="visually-hidden">Перечень причин посетить Седону</h2>
+          <ul class="reasons__list">
+            <li class="reasons__item reasons__item--large city-reasons">
+              <h3 class="reasons__title">Настоящий городок</h3>
+              <p class="reasons__number">&mdash; №1 &mdash;</p>
+              <p class="reasons__description">Седона - не аттракцион для туристов, там течет своя жизнь</p>
+              <ul class="city-reasons__list">
```

Comment:
Я думаю, тебе все-таки придется сделать этот вложенный список еще и элементом блока `reasons`.
Чтобы через него определить его положение внутри элемента `reasons__item`. Можно конечно сказать, что он будет определяться блоком `city-reasons`, но на самом деле в `reasons__item` тоже уже будет сетка.

Я бы предложил так:
```suggestion
            <li class="reasons__item reasons__item--large">
              <h3 class="reasons__title">Настоящий городок</h3>
              <p class="reasons__number">&mdash; №1 &mdash;</p>
              <p class="reasons__description">Седона - не аттракцион для туристов, там течет своя жизнь</p>
              <ul class="reasons__inner-list city-reasons">
```